mep3_simulation/webots_data/controllers/judge/judge.py

```diff
@@ -6,10 +6,10 @@
 
 
 INITIAL_POSE_MATRIX = [
-    ('big', 'purple', [1.249, 0.102, pi/2]),
-    ('small', 'purple', [1.2755, 0.443, pi]),
-    ('big', 'yellow', [-1.249, 0.102, pi/2]),
-    ('small', 'yellow', [-1.2755, 0.443, 0]),
+    ('big', 'purple', [0.102, 1.0, pi/2]),
```

Comment:
Posto ove godine mozemo da pomeramo startne pozicije, ja bih za ovo i u [`mep3_bringup/launch/robot_launch.py`](https://github.com/memristor/mep3/pull/238/files#diff-3fec134d621cd478e3d075c61002e83e980a7afe11d064209c5ec0bd42b3189c) pozicije citao iz parametara kada su definisani.

Reply:
To je odlicna ideja. Mozes dati nekom da sredi u drugom PRu, ovo sada je bas hack